project-base/build-dev.xml

```diff
@@ -54,6 +54,7 @@
             <arg value="translation:extract" />
             <arg value="--bundle=ShopsysShopBundle" />
             <arg value="--dir=${path.src}/Shopsys/ShopBundle" />
+            <arg value="--dir=${path.app}/Resources" />
```

Comment:
what about monorepo `build.xml` ?

Reply:
You are not extending templates in monorepo, so it is not necessary